Makefile

```diff
@@ -40,48 +31,25 @@ $(NIS) setup: package.json
 lint: $(NIS)
 	$(NPM) run lint
 
-build/katex.js: katex.js $(wildcard src/*.js) $(NIS)
-	mkdir -p build
-	$(BROWSERIFY) -t [ babelify ] $< --standalone katex > $@
-
-build/katex.min.js: build/katex.js
-	mkdir -p build
-	$(UGLIFYJS) < $< > $@
-
-build/katex.css: static/katex.less $(wildcard static/*.less) $(NIS)
-	mkdir -p build
-	./node_modules/.bin/lessc $< $@
-
-build/katex.min.css: build/katex.css
+webpack: katex.js $(wildcard src/*.js) $(wildcard static/*.less) $(NIS)
 	mkdir -p build
-	$(CLEANCSS) -o $@ $<
+	mkdir -p build/contrib
+	mkdir -p build/fonts
```

Comment:
Doesn't webpack create the directories it needs if they don't exist?

Reply:
@kevinbarabash Yes, it does. Will remove them.